examples/step-45/step-45.cc

```diff
@@ -14,417 +14,728 @@
  * ---------------------------------------------------------------------
 
  *
- * Author: Markus Buerg, University of Karlsruhe, 2010
+ * Author: Daniel Arndt, Matthias Maier, 2015
+ *
+ * Based on step-22 by Wolfgang Bangerth and Martin Kronbichler
  */
 
+// This example program is a slight modification of step-22 to demonstrate
+// the usage of periodic boundary conditions in deal.II. We thus omit to
+// discuss the majority of the source code and only comment on the parts
+// that deal with periodicity constraints. For the rest have a look at
+// step-22.
 
-// @sect3{Include files}
+// In order to implement periodic boundary conditions only two functions
+// have to be modified:
+// - StokesProblem<dim>::setup_dofs(): To populate a ConstraintMatrix
+//   object with periodicity constraints
+// - StokesProblem<dim>::run (): To supply a distributed triangulation with
```

Comment:
mark up the two function names with `<code>...</code>` -- doxygen doesn't automatically link within tutorials